diff --git a/target_repo/luigi/luigi/task.py b/target_repo/luigi/luigi/task.py
index c090d50..e17753d 100644
--- a/target_repo/luigi/luigi/task.py
+++ b/target_repo/luigi/luigi/task.py
@@ -323,8 +323,7 @@ class Task(object):
         params_str = {}
         params = dict(self.get_params())
         for param_name, param_value in six.iteritems(self.param_kwargs):
-            if params[param_name].significant:
-                params_str[param_name] = params[param_name].serialize(param_value)
+            params_str[param_name] = params[param_name].serialize(param_value)
 
         return params_str
 
